### DISCUSSION TITLE
V14 firgate/reviews more oddness with zone order in end message

### DISCUSSION THREAD

**Discussion by ediguidedog** [blakeblackshear/frigate#12150] [General]
So I'm noticing that in the `end `message the order of the after zones seems to have have changed from the way they were presented in earlier versions. Whereas in my previous version I would see the following in entered zones via the path at the top of the image below.

```
 entered_zones:
        - front path
        - side path
        - doorstep
```
       

In V14 I see 
```
zones:
          - doorstep
          - front_path
          - side_path
```

Putting aside that we read from top to bottom in the first and bottom to top in V14 I think in pre 14 the order is the time the person FIRST entered the zone whereas in V14 it's the LAST time the zone was entered.  The reason this is an issue for me is when I'm home I only want notifications on my front camera if someone arrives at my property and I don't want to know if someone leaves and this is done by checking if either the front door (doorstep) or by the side of the house (side Path) was the first zone touched

The image below will hopefully help 

![image](https://github.com/blakeblackshear/frigate/assets/73383390/b616df3e-2e10-431d-8952-467aba5c01c7)

When I leave I will hit the red or blue zone first so I can ignore but when someone arrives they hit the yellow zone (Front Path) first. Problem is if they arrive in V14 they hit the front path then side path then front path again and then doorstep.  When I look at the end message it looks like the first sight was the side path which is misleading. 

Is this intended or a bug?

**Comment by NickM-27** [comment]
actually, the order is alphabetical right now. I do understand that the events before had them in chronological order so that will be fixed

**Reply by ediguidedog** [reply]
Great thanks
